### DISCUSSION TITLE
facing issue with kong manager Ui , not able to access gateway services

### DISCUSSION THREAD

**Discussion by raghunath03** [Kong/kong#13109] [Help]
Initially i have installed Kong ingress controller 
for that i refered below link
https://docs.konghq.com/kubernetes-ingress-controller/latest/get-started/ 
We have installed Gateway API CRDs before installing KIC
1.kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml

2.Then we have created a Gateway Class using yaml file and applied it manually using kubectl apply command.
 
apiVersion: gateway.networking.k8s.io/v1
kind: GatewayClass
metadata:
  name: kong
  annotations:
    konghq.com/gatewayclass-unmanaged: 'true'

spec:
  controllerName: [konghq.com/kic-gateway-controller](http://konghq.com/kic-gateway-controller)

3.Post creating kong gateway class we have installed KIC and kong gateway using helm commands

helm repo add kong https://charts.konghq.com
helm repo update
helm install kong kong/ingress -n kong --create-namespace 

4.After that i created values-manager.yaml file and added below 
env:
  admin_gui_url: http://manager.example.com
  admin_gui_api_url: http://admin.example.com
  # Change the secret and set cookie_secure to true if using a HTTPS endpoint
  admin_gui_session_conf: '{"secret":"secret","storage":"kong","cookie_secure":false}'
enterprise:
  rbac:
    enabled: true
    admin_gui_auth: basic-auth
manager:
 enabled: true
 http:
   enabled: true
 tls:
   enabled: false
admin:
  enabled: true
  http:
    enabled: true
  tls:
    enabled: false
    
 ingress:
   enabled: true
   hostname: manager.example.com
   path: /
   pathType: Prefix
   ingressClassName: kong
   
   5.helm upgrade kong-cp kong/kong -n kong --values ./values-manager.yaml
   
   when i search on browser using external load balancer ip  i am getting below " gateway service could not be retrived" error 
   
![Screenshot from 2024-05-29 20-25-22](https://github.com/Kong/kong/assets/134500552/d221fa27-8a5c-4f07-b612-3d3d192b7ad9)
what should i do in order resolve above error 
also i am getting below 8001 issue , which i have not enabled/mentioned anywhere
![Screenshot from 2024-05-29 20-26-16](https://github.com/Kong/kong/assets/134500552/786f93b4-52a0-47ff-b169-557d9ef7a601)

![Screenshot from 2024-05-31 10-51-51](https://github.com/Kong/kong/assets/134500552/01608b2e-7dc6-434c-88e8-82b2834b4259)

any input/ suggestion for above ?

**Comment by StarlightIbuki** [comment]
@sumimakito Could you take a look?

**Comment by sumimakito** [comment]
Hello @raghunath03, could you share the `kubectl describe` output with us?

@StarlightIbuki I would also suggest we have members from the K8s team take a look into this as well, if possible.

**Reply by StarlightIbuki** [reply]
@randmonkey Could you take a look?
